claim-check/code-samples/sample-2/client-consumer/ClientConsumer.csproj

```diff
@@ -8,8 +8,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.EventHubs" Version="2.2.1" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="9.4.1" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.6" />
+    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.0.0-preview.6" />
```

Comment:
Quick Note:  There have been some organizational changes between Preview 6 and what will be in the next release.  The consumer client and related types have been moved to the `Azure.Messaging.EventHubs.Consumer` namespace.  Producer-related types moved to `Azure.Messaging.EventHubs.Producer`.

Other than those changes, I don't see anything that I believe would break when moving versions.